lib/hbase/hbase-rest.js

```diff
@@ -42,6 +42,9 @@ var tables = {
     'validations_by_validator',
     'validations_by_date',
     'validator_reports'
+  ],
+  manifests: [
```

Comment:
I think I'd prefer that manifests was just grouped in with the validations tables